serialization.rst

```diff
@@ -167,12 +167,16 @@ possible mapping values are:
 - *not set* or ``NULL`` — this is the default.
 
   - A BSON array will be deserialized as a PHP ``array``.
-  - A BSON document (root or embedded) without ``__pclass`` property [1]_ becomes a
-    PHP ``stdClass`` object, with each BSON document key becoming a
+  - A BSON document (root or embedded) without a ``__pclass`` property [1]_
+    becomes a PHP ``stdClass`` object, with each BSON document key set as a
     public ``stdClass`` property.
-  - A BSON document (root or embedded) with ``__pclass`` property [1]_ becomes
+  - A BSON document (root or embedded) with a ``__pclass`` property [1]_ becomes
     a PHP object of the class name as defined by the ``__pclass`` property.
 
+    If the named class does not exist, ``stdClass`` will be used and each BSON
```

Comment:
It should bail out with an exception instead of silently ignoring the problem. This is likely a programming error, and just returning stdClass, and hence, losing information, is not great. Goes against your round trip arguments too.